snowpack/src/build/process.ts

```diff
@@ -181,9 +177,15 @@ export async function addBuildFilesFromMountpoints(state: BuildState): Promise<v
 
   for (const [mountKey, mountEntry] of Object.entries(config.mount)) {
     logger.debug(`Mounting directory: '${mountKey}' as URL '${mountEntry.url}'`);
-    const files = (await new fdir().withFullPaths().crawl(mountKey).withPromise()) as string[];
-
-    possibleFiles.push(...files);
+    const allMatchedFiles = (await new fdir()
+      .withFullPaths()
+      .crawl(mountKey)
+      .withPromise()) as string[];
+    if (mountEntry.dot) {
+      possibleFiles.push(...allMatchedFiles);
+    } else {
+      possibleFiles.push(...allMatchedFiles.filter((f) => !IS_DOTFILE_REGEX.test(slash(f)))); // TODO: use a file URL instead
```

Comment:
Note: having a RegEx that is conditionally `/` or `\` based on the system is very difficult to predict! Overall we should be using [Node file URLs](https://nodejs.org/api/url.html#url_url_fileurltopath_url) which are predictable, consistent, and always `/`.

Anyways, I converted the filepaths here to always be `/` (which they should be), and made the RegEx always test for the same thing. That was less work than the alternative.